service/service.go

```diff
@@ -54,7 +54,8 @@ type settingsRepository interface {
 
 type releaseRepository interface {
 	CreateRelease(ctx context.Context, r model.Release) error
-	ReadRelease(ctx context.Context, projectID, releaseID uuid.UUID) (model.Release, error)
+	ReadRelease(ctx context.Context, releaseID uuid.UUID) (model.Release, error)
+	ReadReleaseForProject(ctx context.Context, projectID, releaseID uuid.UUID) (model.Release, error)
```

Comment:
This is only temporary. Once the refactoring in #177 is complete, all release repository functions that handle a single release will only require the release ID.